tethys_cli/start_commands.py

```diff
@@ -0,0 +1,134 @@
+        "and scaffold a hello_world app as well.",
+    )
+
+    quickstart_parser.set_defaults(func=quickstart_command)
+
+
+def start_command(args):
+    manage_path = get_manage_path(args)
+    if args.port:
+        primary_process = ["python", manage_path, "runserver", args.port]
+    else:
+        primary_process = ["python", manage_path, "runserver"]
+
+    run_process(primary_process)
+
+
+def quickstart_command(args):
+    """
+    "Start the tethys server. If being done for the first time, this will also generate a portal config file, configure the database, and prompt to scaffold an app as well."
+    """
+    portal_config_args = Namespace(
+        type=GEN_PORTAL_OPTION,
+        directory=None,
+        overwrite=False,
+        tethys_portal_settings={},
+    )
+    portal_config_path = get_destination_path(portal_config_args, check_existence=False)
+    if not os.path.exists(portal_config_path):
+        generate_command(portal_config_args)
+
+    db_config_args = Namespace(
+        command="configure",
+        db_alias="default",
+        username="tethys_default",
+        password="pass",
+        superuser_name="tethys_super",
+        superuser_password="pass",
+        portal_superuser_name="admin",
+        portal_superuser_email="",
+        portal_superuser_password="pass",
+        no_confirmation=False,
+    )
+    db_config_options = process_args(db_config_args)
+    if not os.path.exists(db_config_options["db_name"]):
+        configure_tethys_db(**db_config_options)
+
+    setup_django()
+    installed_apps = get_installed_tethys_items(apps=True)
+    if not installed_apps:
+        # Automatically scaffolds and installs an app
+        app_scaffold_args = Namespace(
+            name="hello_world",
+            extension=False,
+            template="default",
+            use_defaults=True,
+            overwrite=False,
+            prefix=f"{get_tethys_home_dir()}/apps",
+        )
+        scaffold_command(app_scaffold_args)
+        os.chdir(relative_to_tethys_home(f"apps/{APP_PREFIX}-hello_world", as_str=True))
+        app_install_args = Namespace(
+            develop=True,
+            file=None,
+            without_dependencies=False,
+            only_dependencies=False,
+            verbose=False,
+            no_db_sync=False,
+            force_services=False,
+            services_file=None,
+            quiet=True,
+            no_sync_stores=False,
+        )
+        install_command(app_install_args, do_exit=False)
+
+        update_settings_args = Namespace(
+            set_kwargs=[
+                (
+                    "TETHYS_PORTAL_CONFIG",
+                    """
+                    MULTIPLE_APP_MODE: False
+                    STANDALONE_APP: hello_world
+                """,
+                )
+            ]
+        )
+        settings_command(update_settings_args)
+
+    webbrowser.open("http://127.0.0.1:8000/")
```

Comment:
Why not call `webbrowser.open()` after the `start_command`?

Reply:
Because the start_command is a blocking process that starts up the tethys server (i.e. "Quit the server with CTRL-BREAK"). I'll look into if we can inject `webbrowser.open` somewhere inside of the `python <manage_path> runserver`  command.

Reply:
It seems that how it is now is our best option. The browser will open up prematurely to the "This site can't be reached. Localhost refused to connect" page, but within a few seconds once the server finishes starting up it will auto-redirect to the app page. If that seems unacceptable, then I'm not sure what else to do.

Reply:
That makes sense. No action needed.